core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -257,6 +256,20 @@ private TableProperties() {
 
   public static final String OWNER = "owner";
 
+  /**
+   * table format related properties
+   */
+  public static final String TABLE_FORMAT = "table-format";
+  public static final String TABLE_FORMAT_MIXED_ICEBERG = "mixed-iceberg";
+  public static final String TABLE_FORMAT_MIXED_HIVE = "mixed-hive";
+
+  public static final String MIXED_FORMAT_PRIMARY_KEY_FIELDS = "mixed-format.primary-key-fields";
+  public static final String MIXED_FORMAT_TABLE_STORE = "mixed-format.table-store";
+  public static final String MIXED_FORMAT_TABLE_STORE_BASE = "mixed-format.table-store.base";
```

Comment:
Would it be more appropriate to use shorter names for property values, such as `base` and `change`?